vm/chain_config.go

```diff
@@ -31,7 +31,7 @@ func (c *ChainConfig) IsIstanbul(num *big.Int) bool {
 	return isForked(c.IstanbulBlock, num)
 }
 
-// IsGalactica returns whether num is either equal to the Istanbul fork block or greater.
```

Comment:
`Shanghai` is the version name of EVM, should keep it. REF commit https://github.com/vechain/thor/commit/1649781af797839a6d09fda2c54a7c4efe32ed5c